master/reference/calicoctl/resources/networkset.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Network set
+canonical_url: https://docs.projectcalico.org/v3.5/reference/calicoctl/resources/networkset
+---
+
+A network set resource (NetworkSet) represents an arbitrary set of IP subnetworks/CIDRs,
+allowing it to be matched by {{site.prodname}} policy.  Network sets are useful for applying policy to traffic
+coming from (or going to) external, non-{{site.prodname}}, networks.
+
+`NetworkSet` is a namespaced resource. `NetworkSets` in a specific namespace
+only applies to [network policies]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy)
+in that namespace. Two resources are in the same namespace if the `namespace`
+value is set the same on both. (See [GlobalNetworkSet]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/globalnetworkpolicy) for non-namespaced network sets.)
+
+The metadata for each network set includes a set of labels.  When {{site.prodname}} is calculating the set of
+IPs that should match a source/destination selector within a
+[network policy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy) rule, it includes
+the CIDRs from any network sets that match the selector.
+
+> **Important**: Since {{site.prodname}} matches packets based on their source/destination IP addresses,
+> {{site.prodname}} rules may not behave as expected if there is NAT between the {{site.prodname}}-enabled node and the
+> networks listed in a network set.  For example, in Kubernetes, incoming traffic via a service IP is
+> typically SNATed by the kube-proxy before reaching the destination host so {{site.prodname}}'s workload
+> policy will see the kube-proxy's host's IP as the source instead of the real source.
+{: .alert .alert-danger}
+
+For `calicoctl` commands that specify a resource type on the CLI, the following
+aliases are supported (all case insensitive): `networkset`, `networksets`, `netsets`.
+
+### Sample YAML
+
+```yaml
+apiVersion: projectcalico.org/v3
+kind: NetworkSet
+metadata:
+  name: a-name-for-the-set
```

Comment:
I know we do this elsewhere, but I'd like to start using more realistic values here when providing sample yamls.

e.g.

```
name: database-ips
namespace: default
```